### PR TITLE
Make machine-id sources flag a comma-separated list.

### PR DESCRIPTION
**Pull request by rjnagal**
Use "/etc/machine-id,/var/lib/dbus/machine-id" as default.

### CODE REVIEW

File: manager/machine.go [hunk 1]
```diff
@@ -210,12 +210,17 @@ func getTopology(sysFs sysfs.SysFs, cpuinfo string) ([]info.Node, int, error) {
...
 		return ""
 	}
-	return strings.TrimSpace(string(id))
+	for _, file := range strings.Split(*machineIdFilePath, ",") {
+		id, err := ioutil.ReadFile(file)
+		if err == nil {
+			return strings.TrimSpace(string(id))
+		}
+	}
+	glog.Infof("Couldn't collect machine-id from any of the files in %q", *machineIdFilePath)
```
Comment (vishh) [inline manager/machine.go]: nit: Why not report an error instead of info?

**Comment by vishh** [comment]
LGTM. The build is failing on CoreOS.

**Comment by vmarmol** [comment]
LGTM

File: manager/machine.go (hunk 1 above)
Comment (rjnagal) [inline manager/machine.go]: I am not sure if we have machine-id on all distro. It might be expected to fail on some hosts.

**Comment by rjnagal** [comment]
e2e test seems to be flaky. Running it again fixed it.

**Comment by vmarmol** [comment]
Grrr need to tackle the flakes. LGTM

**Pull request merged**